buzzmobile/launch/main.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
+    <param name="hokuyo_node/port" value="/dev/ttyACM0"/>
```

Comment:
Can we have these params that are specific to a node be in the same <tag> as the node itself as an `args`?